lib/assets/SnapshotHelper.swift

```diff
@@ -36,18 +37,31 @@ class Snapshot: NSObject {
         let path = "/tmp/language.txt"
 
         do {
-            locale = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding) as String
-            deviceLanguage = locale.substringToIndex(locale.startIndex.advancedBy(2, limit:locale.endIndex))
-            app.launchArguments += ["-AppleLanguages", "(\(deviceLanguage))", "-AppleLocale", "\"\(locale)\"", "-ui_testing"]
+            deviceLanguage = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding) as String
```

Comment:
@bartoszj Thanks for this!  I did some playing around with this and managed to get thinks working with a smaller change to the helper.  What do you think of this? 

```swift
    class func setLanguage(app: XCUIApplication) {
        let path = "/tmp/language.txt"

        do {
            let input = try NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding) as String
            
            deviceLanguage = input
            locale = input
            
            app.launchArguments += ["-AppleLanguages", "(\(deviceLanguage))", "-AppleLocale", "\"\(locale)\"","-ui_testing"]
        } catch {
            print("Couldn't detect/set language...")
        }
    }
```

I tested with these languages / regions
```ruby
languages([
  "en-US",
  "zh_Hans",
  "zh-Hant",
  "ru",
  "pt",
  "pt-PT",
  "pt-BR"
])
```